.github/workflows/bump-version.yml

```
@@ -8,12 +9,115 @@ on:
+      - name: Run bin/release.php script
+        run: |
+         echo "Running 'php bin/release.php ${{ env.CURRENT_VERSION }} ${{ env.NEW_VERSION }}'"
+         echo "n" | php bin/release.php ${{ env.CURRENT_VERSION }} ${{ env.NEW_VERSION }}
+         if [ $? -ne 0 ]; then
+           echo "Failed to run the release script"
+           exit 1
+         fi
+         git push --set-upstream origin update/wp-parsely-version-to-${{ env.NEW_VERSION }}
+
+      - name: Format the version changelog
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:6: Double quote to prevent globbing and word splitting

(shellcheck)